Documentation/kubernetes-upgrade.md

```diff
@@ -65,9 +65,9 @@ The kube-proxy is run as a "static pod". To upgrade the pod definition, simply m
 ## Example Upgrade Process:
 
 1. Prepare new pod manifests for master nodes
-1. Prepare new pod manifests for worker nodes
-1. For each master node:
+2. Prepare new pod manifests for worker nodes
+3. For each master node:
     1. Back up existing manifests
-    1. Update manifests
-1. Repeat same steps above for each worker node
+    2. Update manifests
+4. Repeat item 3 for each worker node
```

Comment:
It will still render correctly regardless though, right (at the cost of possibly getting out of sync with rendered numbering)?  I guess there's sort of precedent in HTML's use of generic "numbered/bulleted list item" notation.  Do we have a project doc-markup style guide?

Will fix and push in a bit.